ambari/src/test/java/io/brooklyn/ambari/AmbariBlueprintLiveTest.java

```diff
@@ -76,47 +76,44 @@ public void tearDown() throws Exception {
 
         return new Object[][]{
                 new Object[]{
-                        "ambari-cluster.yaml", "softlayer", "ams01", "UbuntuOnSoftlayer", MutableMap.<String, String>builder()
-                        .put("minRam", "16384")
-                        .put("minCores", "4")
-                        .put("osFamily", "ubuntu")
-                        .put("osVersionRegex", "12.*")
+                        "ambari-cluster.yaml", "aws-ec2", "eu-central-1", "UbuntuOnAWS", MutableMap.<String, String>builder()
+                        .put("hardwareId", "m4.large")
+                        .put("imageId", "eu-central-1/ami-142ded7b")
                         .put("stopIptables", "true")
                         .build()
                 },
                 new Object[]{
-                        "ambari-cluster.yaml", "softlayer", "ams03", "CentOSOnSoftlayer", MutableMap.<String, String>builder()
-                        .put("minRam", "16384")
-                        .put("minCores", "4")
-                        .put("osFamily", "centos")
-                        .put("osVersionRegex", "6.*")
+                        "ambari-cluster.yaml", "aws-ec2", "eu-central-1", "CentOSOnAWS", MutableMap.<String, String>builder()
+                        .put("hardwareId", "m4.large")
+                        .put("imageId", "eu-central-1/ami-14211709")
                         .put("stopIptables", "true")
                         .build()
                 },
                 new Object[]{
-                        "ambari-cluster.yaml", "softlayer", "lon02", "RHEDOnSoftlayer", MutableMap.<String, String>builder()
-                        .put("minRam", "16384")
-                        .put("minCores", "4")
-                        .put("osFamily", "rhel")
-                        .put("osVersionRegex", "6.*")
+                        "ambari-cluster.yaml", "aws-ec2", "eu-central-1", "RHELOnAWS", MutableMap.<String, String>builder()
+                        .put("hardwareId", "m4.large")
+                        .put("imageId", "eu-central-1/ami-cd362ca1")
```

Comment:
While I would agree that the blueprint should work without requiring a user to set an image id I lean towards having image Ids in live tests otherwise you risk tests that fail intermittently due to the image brooklyn picks.  It can also be difficult to work out from the logs what image was picked so replicating failures can be easier.